src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/DSA.cs

```diff
@@ -35,9 +35,20 @@ public abstract partial class DSA : AsymmetricAlgorithm
             return (DSA?)CryptoConfig.CreateFromName(algName);
         }
 
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("maccatalyst")]
+        public static new DSA Create()
+        {
+            return CreateCore();
+        }
```

Comment:
The indirection is necessary because of the `UnsupportedOSPlatform` attributes which would otherwise have to be in 5 different places.